src/applications/edu-benefits/10203/tests/containers/IntroductionPage.unit.spec.jsx

```diff
@@ -21,10 +23,49 @@ describe('Edu 10203 <IntroductionPage>', () => {
         }}
       />,
     );
+
     expect(tree.find('FormTitle').props().title).to.contain('Apply');
     expect(tree.find('withRouter(Connect(SaveInProgressIntro))').exists()).to.be
       .true;
     expect(tree.find('.process-step').length).to.equal(5);
     tree.unmount();
   });
+
+  it('should show entitlement remaining alert', () => {
+    const remainingEntitlementGreaterThan180 = {
+      remainingEntitlement: { months: 6, days: 3 },
+    };
+
+    const tree = shallow(
+      <IntroductionPage
+        isLoggedIn
+        remainingEntitlement={remainingEntitlementGreaterThan180}
+        getRemainingEntitlement={getRemainingEntitlement}
```

Comment:
You don't need to use the actual action here (in fact it's better if you do not since you want to test the component by itself).